config/initializers/devise.rb

```diff
@@ -12,7 +12,7 @@
   # The secret key used by Devise. Devise uses this key to generate
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
-  config.secret_key = Settings.devise.secret_key
+  config.secret_key = Rails.application.secrets.devise.secret_key
```

Comment:
This is clearly broken when the devise key does not exist, but I'm not sure what makes sense to do here. Should this key always exist?